capture.py

```diff
@@ -416,6 +421,17 @@ def _isolated_nodes(nodes, panel):
             cmds.isolateSelect(panel, addDagObject=obj)
     yield
 
+@contextlib.contextmanager
+def _maintained_time():
+    """Context manager for preserving (resetting) the time after the context"""
+    from maya import cmds
```

Comment:
Or sorry, I'm referring to line 427 specifically, not that you re-used the name of the manager. It looks great.